docs/server-admin.md

```diff
@@ -0,0 +1,37 @@
+# The administration panel
+
+Iodide has an administration view under `/admin`, based on the [Django
+admin](https://docs.djangoproject.com/en/2.1/ref/contrib/admin/) interface.
+
+To log into this interface, a user must either have the `is_staff` or `is_superuser` permission set. Most operations (including the one currently
```

Comment:
This wasn't true for me. I get a *Permission Denied* error when only `is_superuser` is set.

Here's what I'm seeing in more detail:

|State|Result|
|---|---|
|Only `is_superuser` is set to true|*You are authenticated as openjck, but are not authorized to access this page. Would you like to login to a different account?*|
|Only `is_staff` is set to true|I can see the admin interface and the following links are present: *Users*, *Notebooks*|
|Both are set to true|I can see the admin interface and the following links are present: *Tokens*, *Users*, *Notebooks*, *Associations*, *Nonces*, *User social auths*|

Reply:
Ah ok, I guess you need is_staff permission to use the admin interface at all. I'll update the docs.